specification/iothub/resource-manager/Microsoft.Devices/2017-07-01/iothub.json

```diff
@@ -1013,9 +881,359 @@
         },
         "deprecated": false
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Devices/IotHubs/{resourceName}/certificates": {
+      "get": {
+        "tags": [ "Certificates" ],
+        "summary": "Get the certificate list.",
+        "description": "Returns the list of certificates.",
+        "operationId": "Certificates_ListByIoTHub",
```

Comment:
should be Certificates_ListByIotHub, autorest generated sdks are getting confused.